API.md

```diff
@@ -10,15 +10,21 @@ const Log = require('ipfs-log')
 
 ### Constructor
 
-#### new Log(ipfs, [id])
+#### new Log(ipfs, access, identity, [id])
 
 Create a log. Each log gets a unique ID passed as an argument. Returns a `Log` instance.
 
 ```javascript
-const log = new Log(ipfs, 'logid')
+const { AccessController, IdentityProvider } = Log
+const keystore = require('orbit-db-keystore').create('./keystorepath')
+const identity = await IdentityProvider.createIdentity(keystore, 'peerid')
+const ipfs = new IPFS()
+const access = new AccessController()
+
+const log = new Log(ipfs, access, identity, 'logid')
 ```
 
-`ipfs` is an instance of IPFS. `id` is a unique log identifier. Usually this should be a user id or similar.
+`ipfs` is an instance of IPFS. `access` is an instance of `AccessController`, by default allows any one to append to the log. `identity` is an instance of [Identity](https://github.com/orbitdb/orbit-db-identity-provider/blob/master/src/identity.js), used to sign entries. `id` is a unique log identifier. Usually this should be a user id or similar.
```

Comment:
"...`AccessController`, which by default..."

Would prolly make sense to add a link to the default ac source file for quick look. Which in turn makes me think we should explain the access controller logic in more detail as to how it can be used to create "custom ac policies". I understand this may be hard to put together as a comprehensive guide, but I think it'd help users if we had at least some explanation of how to use it.